src/core/compact_object.h

```diff
@@ -479,6 +484,7 @@ class CompactObj {
     // using 'packed' to reduce alignement of U to 1.
     JsonWrapper json_obj __attribute__((packed));
     SBF* sbf __attribute__((packed));
+    TopKeys* topk __attribute__((packed));
```

Comment:
We use a different ds than valkey, I will write tests but probably we won't be 1-1 do the the probabilistic nature of the ds

Reply:
does valkey support it?

Reply:
When I said valkey I meant the one who can't be named :smile: